axlearn/experiments/text/gpt/common.py

```diff
@@ -267,12 +269,17 @@ def model_config(
         batch_axis_names=batch_axis_names,
         seq_axis_names="seq",
     )
+
+    device_platform = np.asarray(jax.devices())[0].platform
```

Comment:
`jax.devices()` during config building may be an unexpected dependency on global state -- should we take a `platform` arg or similar?

Reply:
We could change it, but I followed the pattern already used here https://github.com/apple/axlearn/blob/89c6f758d4b54f78273c2b4b40c16923b7416daa/axlearn/common/utils.py#L1231